sdk/src/error.rs

```diff
@@ -288,10 +288,6 @@ pub enum Error {
     #[error("could not acquire OpenSSL FFI mutex")]
     OpenSslMutexError,
 
```

Comment:
There are probably more error codes I could remove. Will do that in a subsequent PR.